pkg/internal/checkup/checkup_test.go

```diff
@@ -424,12 +424,19 @@ type executorStub struct {
 	executeErr error
 }
 
+func mockSuccessfulResults() status.Results {
+	const sentPackets int64 = 10
+	return status.Results{
+		TrafficGenSentPackets:      sentPackets,
+		VMUnderTestReceivedPackets: sentPackets,
+	}
+}
+
 func (es executorStub) Execute(_ context.Context, vmiUnderTestName, trafficGenVMIName string) (status.Results, error) {
 	if es.executeErr != nil {
 		return status.Results{}, es.executeErr
 	}
-
-	return status.Results{}, nil
+	return mockSuccessfulResults(), nil
```

Comment:
Please consider adding a `returedResults` to the `executorStub`, so you could control the returned results from outside.

Reply:
DONE